api.js

```diff
@@ -898,7 +898,9 @@ API.prototype.reference = function(options) {
   assert(options,         "Options is required");
   assert(options.baseUrl, "A 'baseUrl' must be provided");
   var reference = {
-    version:            '0.2.0',
+    version:            0,
+    '$schema':          'http://schemas.taskcluster.net/base/v1/' +
+                        'api-reference.json',
```

Comment:
In other schema references we typically suffix the url with a '#' character (maybe to denote that the root of the schema at this url is the schema to validate against?), e.g. 'http://schemas.taskcluster.net/base/v1/api-reference.json#' - should we do this here too?